js/src/views/Web/web.js

```diff
@@ -98,18 +103,23 @@ export default class Web extends Component {
     });
   };
 
-  handleOnRefresh = (ev) => {
+  onRefresh = () => {
     const { displayedUrl } = this.state;
-    const hasQuery = displayedUrl.indexOf('?') > 0;
-    const separator = hasQuery ? '&' : '?';
+
+    // Insert timestamp
+    // This is a hack to prevent caching.
+    const parsed = parseUrl(displayedUrl);
+    parsed.query = parseQuery(parsed.query);
+    parsed.query.t = Date.now().toString();
+    delete parsed.search;
 
     this.setState({
       isLoading: true,
-      url: `${displayedUrl}${separator}t=${Date.now()}`
+      url: formatUrl(parsed)
     });
   };
 
-  handleIframeLoad = (ev) => {
+  iframeOnLoad = () => {
```

Comment:
But it is a `load` even *on* the `iframe`, not a `iframeLoad` event. I think the current form is more expressive.